Dockerfile

```diff
@@ -1,26 +1,26 @@
-FROM alpine:3.7
+FROM python:3.8.2-slim
 LABEL maintainer="smartlab-dev@mpt.mp.br"
 
+ENV PYTHONPATH /app:/usr/lib/python3.8/site-packages
+
 COPY requirements.txt /app/requirements.txt
 COPY app/*.py /app/
 COPY uwsgi.ini /etc/uwsgi/conf.d/
 COPY start.sh /start.sh
 
-RUN apk --update --no-cache add build-base libffi-dev openssl-dev python3-dev libffi openssl ca-certificates python3 cyrus-sasl-dev libstdc++ uwsgi-python3 gfortran openblas-dev && \
-    pip3 install --upgrade pip setuptools && \
+WORKDIR /app
+
+# Moved commented out packages to flask-dataviz
+RUN apt-get update && \
+    apt-get install -y build-essential uwsgi-plugin-python3 && \
     pip3 install -r /app/requirements.txt && \
-    apk del build-base libffi-dev openssl-dev python3-dev libffi openssl ca-certificates && \
-    rm -rf /var/cache/apk/* && \
-    rm -rf ~/.cache/ && \
     mkdir -p /var/run/flask && \
-    chown -R uwsgi:uwsgi /var/run/flask /app /etc/uwsgi/conf.d 
+    apt-get clean
 
 ENV LANG C.UTF-8
 ENV DEBUG 0
 ENV FLASK_APP /app/main.py
```

Comment:
I think that a `USER wsgi` is missing here...

Reply:
Will do both USER and chown.